ksql-rest-app/src/main/java/io/confluent/ksql/rest/entity/SourceDescription.java

```diff
@@ -38,45 +40,120 @@
 public class SourceDescription extends KsqlEntity {
 
   private final String name;
+  private final String queries;
   private final  List<FieldSchemaInfo> schema;
-  private final DataSource.DataSourceType type;
+  private final String type;
```

Comment:
Why do we have to change this to string?

Reply:
@hjafarpour because it was returning a Query which is not a valid value on the enum